src/pages/publication/my_publications/PublicationList.tsx

```diff
@@ -46,6 +51,17 @@ const PublicationList: FC<PublicationListProps> = ({ publications }) => {
   const [openModal, setOpenModal] = useState(false);
   const [deletePublicationId, setDeletePublicationId] = useState('');
   const [deletePublicationTitle, setDeletePublicationTitle] = useState('');
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const handleChangePage = (_: any, newPage: number) => {
```

Comment:
kunne lagt til type for eventen her også, når den først er importert

Reply:
i mui-eksempelet er typen `unknown` 😝 vet ikke om det er så mye bedre

Reply:
nvm, fant typen i et annet eksempel :)

Reply:
åja, antok egentlig at dette også var ChangeEvent som den under